supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -525,22 +523,22 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
         val thisUnexpectedRetry = ur + 1
         if (thisUnexpectedRetry <= maxUnexpectedRetries) {
           // Increment unexpected retry count and preemption count stays the same
-          writeFuturePreemptedAndUnexpectedRetryCounts(p, thisUnexpectedRetry).map { _ =>
```

Comment:
OOC: why the change about not waiting anymore for the result of `writeFuturePreemptedAndUnexpectedRetryCounts` ?

Reply:
Futures are back for the future.